herbie/settings.py

```diff
@@ -57,7 +57,9 @@
     'rest_framework',
     'herbieapp.apps.HerbieappConfig',
     'rest_framework.authtoken',
-    'social_django'
+    'social_django',
+    'dependency_injector',
+    'inject'
```

Comment:
Are we using two different DI Libraries here?

Reply:
not really, first implementation was with the `dependency_injector`, creating instances with providers. But afterwards I found this second library `inject` , that is more simple one and it is much easier to dynamically inject dependencies to commands and controllers. I left "dependency_injector" package to show the difference of creating instances of objects for example in class `ControllerDependencyProvider`

    # simple instantiation of class
    permission_classes_provider = IsAuthenticated()

    # Using dependency injector package
    # permission_classes_provider = providers.Singleton(IsAuthenticated)